libsolidity/codegen/CompilerContext.cpp

```diff
@@ -32,6 +32,7 @@
 #include <libyul/AsmPrinter.h>
 #include <libyul/AsmAnalysis.h>
 #include <libyul/AsmAnalysisInfo.h>
+#include <libyul/AST.h>
```

Comment:
Same here

Reply:
In all theses cases the files relied on on it being implicitly imported via `AsmParser.h`. I've changed `AsmParser.h` to only include the forward declaration.